### ISSUE TITLE
Emergency one: InventorySwitcher deletes inventory on WorldSwitch after /bbox reload

### ISSUE THREAD

**Issue by wellnesscookie** [BentoBoxWorld/InvSwitcher#7]
**Description**
InventorySwitcher will not work correctly after using /bbox reload

**Steps to reproduce the behavior:**
- Have InventorySwitcher installed
- Have items in inventory
- Do /bbox reload
- Switch to any other world
- Have your items not transferred (Inventory deleted on every next world switch)

**Solution to fix:**
- Restart your server
- Works!

**Expected behavior**
- Work after reload, it was a bad hit for us since peak days are present and 90/110 (online at the moment) players were affected :(

**Server Information:**

 - Database being used: **mySQL** / **JSON**
 - OS: **Linux GNU Debian 9** / **Windows 10**
 - Java Version: **Java 10** / **Java 8**
 - BentoBox version: **1.5.3** / **1.5.3**
 - Addons installed:
```
[16:12:17 INFO]: Running SPIGOT 1.13.2.
[16:12:17 INFO]: BentoBox version: 1.5.3
[16:12:17 INFO]: Loaded Game Worlds:
[16:12:17 INFO]: commands.bentobox.version.game-world
[16:12:17 INFO]: Loaded Addons:
[16:12:17 INFO]: BentoBox-InvSwitcher 0.0.3-SNAPSHOT
[16:12:17 INFO]: BSkyBlock 1.5.0
[16:12:17 INFO]: DimensionalTrees 1.1 - SerbCraft
[16:12:17 INFO]: SerbCraftAddon 2.0
```
 - Other plugins:
`[16:12:29 INFO]: Plugins (3): Relative_Coordculator, BentoBox, CrateReloaded`

**Additional context**
After this had happened, I immediately went to start my test server and try to replicate this - and yes, it happens after execution of /bbox reload command.
I tagged it as Emergency, as I don't wish for any other server that has players on it to lose their inventory after this.

**Comment by wellnesscookie** [comment]
The data in .json is filled with nulls of course.
The one and only file on my server: https://pastebin.com/G8pd07cw

**Comment by tastybento** [comment]
~~Hmm, I can't replicate this yet. I'm using JSON database. I'll try with MYSQL.~~
~~Can you still replicate on your test server? Do you see any errors in the console?~~

Replicated on both. I had to change worlds more than once to get it to remove the inventory.

**Comment by tastybento** [comment]
Okay, this was fixed already on 0.0.4-SNAPSHOT. It was due to me not registering the teleport listener with BentoBox.
